mikidown/config.py

```diff
@@ -16,6 +15,9 @@
     " 'module' object has no attribute 'makeExtension'"
 )
 
+## @pedro says.. this is out "definitive version".. and a pain cos it should be in __init__
+## othersie whole module has to be loaded
+
```

Comment:
This I completely agree with. It's part of a refactor that's escaped my mind due to aforementioned IRL busies

Normally when I set up projects, the __version_info__ and other fancy stuff in a minimal __init__ that describes module metadata. The original code had the __version__ stuck in here.